templates/_footer.phtml

```diff
@@ -3,7 +3,11 @@
                 <div class="col-md-12">
                     <hr/>
                     <p>
-                        Copyright &copy; <?php echo esc(date("Y")); ?> The Godot Engine community - MIT licensed
+                        Copyright &copy; <?php echo esc(date("Y")); ?> The Godot Engine community - MIT licensed 
```

Comment:
Useless trailing space in the diff. There is an [.editorconfig](https://github.com/godotengine/godot-asset-library/blob/master/.editorconfig) file in the project, which your editor should be able to understand, see [editorconfig.org](https://editorconfig.org/#download)